src/edit/index.vue

```diff
@@ -60,6 +64,11 @@ export default {
     isReadyToUpload() {
       const { token, uploadUrl } = this.element.data;
       return token && this.file && uploadUrl;
+    },
+    isUnfocus: ({ isDisabled, isFocused }) => !isDisabled && !isFocused,
```

Comment:
This name sounds a bit weird to me because:
- _unfocus_ sounds like an action
- we already have `isFocused` so the name doesn't reflect that it's not a negated value of it

We could negate the condition and rename it to something like `isFocusedOrDisabled` to solve that?

Reply:
Yeah, I agree